dotgov-domains/current-federal.csv

```diff
@@ -569,7 +524,7 @@ USASEANCONNECT.GOV,Federal Agency,Department of State,Jakarta,Jakarta
 USCONSULATE.GOV,Federal Agency,Department of State,Washington,DC
 USEMBASSY.GOV,Federal Agency,Department of State,Washington,DC
 USMISSION.GOV,Federal Agency,Department of State,Washington,DC
-STATEOIG.GOV,Federal Agency,Department of State OIG,Arlington,VA
+STATEOIG.GOV,Federal Agency,"Department of State, Office of Inspector General",Arlington,VA

```

Comment:
I'm expecting that we'll get this moved into `Department of State` after we publish the `Organization` field, and can specify the OIG as the organization there.